src/elements/fields/TextField.jsx

```diff
@@ -89,6 +90,15 @@ function TextField({
 }) {
   const servar = element.servar;
   const inputType = fieldProps.as === 'textarea' ? 'textarea' : 'input';
+
+  useHotkeys(
+    'enter',
+    (e) => (inputType === 'textarea' ? e.stopPropagation() : null),
```

Comment:
can this instead be inputType === 'textarea' && e.stopPropagation()

Reply:
also have you tested that enter successfully creates newlines for textareas on forms and won't attemp to submit that step? but will submit for text fields?

Reply:
Done. Changed the logic to use && operator. Yes I have tested the key event and validated that it behaves as expected in the text area